src/main/java/org/ow2/proactive/procci/model/occi/infrastructure/mixin/Credentials.java

```diff
@@ -33,7 +33,8 @@
      */
     public Credentials(String publickey, List<Entity> entities) {
```

Comment:
Why not using Lombok for getter and setter in this class? Would be convenient to be consistent.